plugins/scaffolder-backend/src/service/helpers.ts

```diff
@@ -107,3 +107,23 @@ export async function findTemplate(options: {
 
   return template as TemplateEntityV1beta3;
 }
+
+/**
+ * Takes a single unknown parameter and makes sure that it's a single string or
+ * an array of strings, and returns as an array.
+ */
+export function parseStringsParam(
+  param: unknown,
+  ctx: string,
```

Comment:
Maybe to name it as `parameterName` to make it easier to understand without looking up for the usage to get what is `ctx`?

Reply:
This is a nice copy-paste from the catalog, but yes, it's not that clear